index.bs

```diff
@@ -2120,7 +2165,7 @@ spec:css-syntax-3;
         value is "{{Credential/[[discovery]]/credential store}}".
       </div>
 
-  4.  Extend {{CredentialRequestOptions}} with the options the new credential type needs to respond
+  5.  Extend {{CredentialRequestOptions}} with the options the new credential type needs to respond
```

Comment:
This is easier if you use `1.` for all the list items. :)

```suggestion
  1.  Extend {{CredentialRequestOptions}} with the options the new credential type needs to respond
```

Reply:
You know, I *liked* the numbers. But they are a pain to maintain. Changed to `1` through the algorithms I touched up on this PR.